specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/examples/AgentPoolsCreate_Flex.json

```diff
@@ -0,0 +1,142 @@
+        ],
+        "type": "Flex",
+        "flexProfile": {
+          "scaleProfile": {
+            "manual": {
+              "machines": [
+                {
+                  "sizes": [
+                    "Dv3",
+                    "Dsv3"
+                  ],
+                  "count": 5
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedClusters/clustername1/agentPools/agentpool1",
+        "type": "Microsoft.ContainerService/managedClusters/agentPools",
+        "name": "agentpool1",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "orchestratorVersion": "1.9.6",
+          "currentOrchestratorVersion": "1.9.6",
+          "maxPods": 110,
+          "osType": "Linux",
+          "tags": {
+            "name1": "val1"
+          },
+          "nodeLabels": {
+            "key1": "val1"
+          },
+          "nodeTaints": [
+            "Key1=Value1:NoSchedule"
+          ],
+          "type": "Flex",
+          "flexProfile": {
+            "scaleProfile": {
+              "manual": {
+                "machines": [
+                  {
+                    "sizes": [
+                      "Dv3",
+                      "Dsv3"
+                    ],
+                    "count": 5
+                  }
+                ]
+              }
+            }
+          },
+          "flexNodesStatus": [
+            {
+              "sizes": "Dsv3",
+              "count": 3
+            },
+            {
+              "sizes": "Dv3",
+              "count": 2
+            }
+          ],
+          "nodeImageVersion": "AKSUbuntu-1804gen2containerd-2021.09.11"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedClusters/clustername1/agentPools/agentpool1",
+        "type": "Microsoft.ContainerService/managedClusters/agentPools",
+        "name": "agentpool1",
+        "properties": {
+          "provisioningState": "Creating",
+          "orchestratorVersion": "1.9.6",
+          "currentOrchestratorVersion": "1.9.6",
+          "maxPods": 110,
+          "osType": "Linux",
+          "tags": {
+            "name1": "val1"
+          },
+          "nodeLabels": {
+            "key1": "val1"
+          },
+          "nodeTaints": [
+            "Key1=Value1:NoSchedule"
+          ],
+          "type": "Flex",
+          "flexProfile": {
+            "scaleProfile": {
+              "manual": {
+                "machines": [
+                  {
+                    "sizes": [
+                      "Dv3",
+                      "Dsv3"
+                    ],
+                    "count": 5
+                  }
+                ]
+              }
+            }
+          },
+          "flexNodesStatus": [
```

Comment:
Should this be in the `flexProfile` rather than alongside it?

Reply:
I think we want profile to be spec and status to be status so put it alongside.

Reply:
I'm fine with that